easybuild/tools/options.py

```diff
@@ -199,8 +201,8 @@ def override_options(self):
             'skip-test-cases': ("Skip running test cases", None, 'store_true', False, 't'),
             'umask': ("umask to use (e.g. '022'); non-user write permissions on install directories are removed",
                       None, 'store', None),
-            'optarch': ("Set architecture optimization, overriding native architecture optimizations",
-                        None, 'store', None),
+            'update-modules-tool-cache': ("Update modules tool cache file(s) after generating module file",
```

Comment:
in general, yes, although with #1177 we will try and add support to update the cache via EB too, I'll look into detailed documentation on this then

for now, mentioning in the release notes that updating the *user* cache by default requires using `--update-modules-tool-cache` should be sufficient

that's a breaking change, so it makes sense to include this for EasyBuild v2.0